musicname/zs_starryhaven_skies_v2.cfg

```
@@ -0,0 +1,9 @@
+"music"
+{
+    "shootingstarsummitremastered.mp3"      "Paper Mario OST - Shooting Star Summit (Remastered)"
+    "songwave2.mp3"                         "FF IX OST - Esto Gaza"
```

I assume this is Final Fantasy 9?
```suggestion
    "songwave2.mp3"                         "FFIX OST - Esto Gaza"
```